### PR TITLE
Solved Issue #7

### PR DESCRIPTION
**Pull request by saikeshari**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by saikeshari** [comment]
Made the Leaderboard section with static data in the github_details.yml file

File: _includes/leaderboard.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<h2>
+  LeaderBoard
+</h2>
+<!-- <h2 class="label">Top Contributors</h2>
+<div class= "center-content">
+  <div class="card card-leaderboard">
+    <div class="card-body">
```
Comment (grace-omotoso) [inline _includes/leaderboard.html]: remove commented code

File: _data/github_details.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  username2git: https://github.com/PriyaBihani
+  PRs: 0
+
+- rank: 5
+  team: 4
+  username1: Akshita Dixit
+  username2: Shubham Bhardwaj
+  username1git: https://github.com/akshitadixit
+  username2git: https://github.com/Shubh4m-B
+  PRs: 0
```
Comment (grace-omotoso) [inline _data/github_details.yml]: Don't hardcode. Use GitHub API to get the number of PRs on the repository based on a fellow's GitHub username

**Review by grace-omotoso — CHANGES_REQUESTED** [review]

File: _includes/leaderboard.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      console.log(teams);
+      temp.innerHTML = vara;
+      console.log(response.data);
+    })
+    .catch(error => console.log(error));
+  }
+
+  function fetchAllPRs() {
+    teams.map((team,index) => {
+      // console.log(team.username + " " + index);
```
Comment (grace-omotoso) [inline _includes/leaderboard.html]: remove commented code

File: _includes/leaderboard.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  {
+    team: 6,
+    username: "saikeshari",
+    count: 0
+  }
+]
+
+
+  const fetchGithubStats = (index,team) => {
+    // console.log(team.username1);
```
Comment (grace-omotoso) [inline _includes/leaderboard.html]: remove

File: _data/github_details.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  team: 2
+  username1: Ojuswi Rastogi
+  username2: Gaurav Kumar
+  username1git: https://github.com/OjusWiZard
+  username2git: https://github.com/GrayFlash
+  PRs: 0
+
+- rank: 4
+  team: 3
+  username1: Adarsh Naidu
```
Comment (grace-omotoso) [inline _data/github_details.yml]: Do we still need this since you are no longer hardcoding? Please, remove if no longer needed

**Review by grace-omotoso — CHANGES_REQUESTED** [review]

**Review by grace-omotoso — APPROVED** [review]

**Comment by grace-omotoso** [comment]
@saikeshari looks good to me but I have one more ask. Can you move it above the Projects section and below the Meet the Pod section? Thanks

**Comment by saikeshari** [comment]
Done!

**Review by grace-omotoso — APPROVED** [review]

**Pull request merged**